ci-operator/step-registry/openstack/provision/bastionproxy/openstack-provision-bastionproxy-commands.sh

```diff
@@ -4,12 +4,12 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-if [[ "$CONFIG_TYPE" != "proxy" ]]; then
+if [[ "$CONFIG_TYPE" != **"proxy"** ]]; then
```

Comment:
Why the double stars? Is it a bash thing?

Reply:
it's a typo probably